scripts/github-actions/triage/comment_workflow.js

```diff
@@ -0,0 +1,187 @@
+async function handleComment(github, context) {
+    
+    // Only continue if the payload was triggered by a comment event and the comment was made by a human user
+    if (!context.payload.comment || context.payload.comment.user.type === "Bot") {
+      console.log("Payload does not contain a comment event made by a human user.");
+      return;
+    };
+    
+    // Get the details of the issue or pull request that triggered the workflow
+    const { issue, issue: { pull_request } } = context.payload;
+    const issueOrPullRequest = pull_request ? { ...issue, type: "pullRequest" } : { ...issue, type: "issue" };
+
+    const isMemberQuery = `query ($login: String!, $org: String!) {
+        user(login: $login) {
+          organization(login: $org) {
+            viewerIsAMember
+          }
+        }
+      }`;
+    const isMemberVariables = {
+        login: context.payload.comment.user.login,
+        org: context.payload.organization.login
+    };
+    
+    const isMemberResult = await github.graphql(isMemberQuery, isMemberVariables);
+
+    let isCommentFromMember = false;
+
+    if (isMemberResult.user.organization != null) {
+        isCommentFromMember = true;
+        console.log(`Comment came from a member of the Org.  isCommentFromMember var: ${isCommentFromMember}`);
+        
+        //Don't do anything if the comment is from a member of the Org
+        return
+    };
+
+    
+    
+    // If comment is from someone outside of the org
+    if (!isCommentFromMember) {
```

Comment:
We return early above when it is a comment from a member, we this if check can be removed.

Reply:
It could be removed, but I prefer it be there for clarity's sake.  There is also the chance that we change the behavior for internal comments in the near future and I don't want folks to know they have to rewrap the rest of the file.

Reply:
Gottchya, thoughts returning fast then? When I see a if like this I scan the entire file first to understand what the default behavior is meant to be.

Reply:
I'm not sure I understand what you are asking?  We need to do the logic before it to determine if the comment creator is a member or not